.ci/kitchen-macosxhighsierra-py2

```diff
@@ -14,6 +14,9 @@ def golden_images_branch = 'master'
 def nox_passthrough_opts = ''
 def concurrent_builds = 1
 def jenkins_slave_label = 'kitchen-slave-mac'
+def kitchen_driver_file = '/var/jenkins/workspace/driver.yml'
+def kitchen_verifier_file = '/var/jenkins/workspace/nox-verifier.yml'
+def kitchen_platforms_file = '/var/jenkins/workspace/pre-golden-nox-platforms.yml'
```

Comment:
why not call it nox-platforms.yml so it's the same as the others?  just wondering.

Reply:
We already have that one setup in sre-jenkins, but for proper MacOS golden images.

Reply:
Just named it differently so that proper golden images can continue without affecting this switch to nox.

Reply:
what is different between the files?

Reply:
Proper golden images, [this](https://github.com/saltstack/sre-jenkins/blob/develop/states/services/jenkins-node/mac-install.sls#L457-L461), pre golden images [this](https://github.com/saltstack/sre-jenkins/blob/develop/states/services/jenkins-node/mac-install.sls#L540-L541) and [this](https://github.com/saltstack/sre-jenkins/blob/develop/states/services/jenkins-node/mac-install.sls#L566), as an example.

Reply:
nice